src/routes/index.js

```diff
@@ -2,18 +2,34 @@ import { Routes, Route } from 'react-router-dom';
 
 import Home from 'pages/Home';
 import LoginPage from 'pages/LoginPage';
+import AuthGuard from 'guards/authGuard';
+import RegisterPage from 'pages/RegisterPage';
+import LoggedInGuard from 'guards/loggedInGuard';
 
 const AppRoutes = () => {
     return (
        <>
             <Routes>
                 <Route 
                     path="/"
-                    element={<Home/>}
+                    element={
+                        <AuthGuard>
```

Comment:
Would you like to wrap all the app routes into <AuthGuard> like this? Is there a possibility to wrap only onces, to make this Routes clear? DRY rule